### ISSUE TITLE
Support Prisma Model Conversion to Other Class Model (C#, Dart, Java, Kotlin, Python, Swift, Typescript)

### ISSUE THREAD

**Issue by sandbox-apps** [prisma/studio#740]
## Problem

TLDR: Model to Class approach much like JSON to something converter approach

Problems are ever changing, so does requirements, so does development, so does implementations like database schema.
Typically there are separate team for front end (web, mobile) and backend...
One typical problem is syncing with changes and updates.

Maybe we could solve this by introspecting the `schema.prisma` **models** and we can get a counter output for the selected language. So that developers are responsible for syncing things to this approach...

Maybe we could implement this inside prisma studio
https://app.quicktype.io/

Your input JSON
![image](https://user-images.githubusercontent.com/86398599/125667083-ecc7fb6d-c21c-43c9-afad-74045aec8511.png)

Output depends on the language configuration
![image](https://user-images.githubusercontent.com/86398599/125667139-85b84c32-d5ed-4f3a-a346-33cfff51d5ce.png)

**Comment by madebysid** [comment]
@sandbox-apps Honestly, this sounds a little out of scope for what Studio is supposed to be currently. I think this is such a subjective thing that it'll be hard to come up with something that works for everyone.

Do you have any thoughts on this @martzoukos?

**Comment by janpio** [comment]
This sounds more like a general request for the Prisma ecosystem. The "Prisma way" to do this would probably be to define your models in Prisma schema language, and then have a custom generator (like e.g. https://www.prisma.io/docs/concepts/components/prisma-schema/generators#community-generators) that converts the structure into whatever you want it to be. Maybe take a look at some of these generators, and if that could already be what you are looking for @sandbox-apps

**Comment by sandbox-apps** [comment]
Thanks for pointing it out I didnt know this community generators exist... This should make my overall backend development happy :D
